tests/ApcuTest.php

```diff
@@ -5,12 +5,12 @@
 use iFixit\Matryoshka;
 
 class ApcuTest extends AbstractBackendTest {
-   protected function setUp() {
```

Comment:
You updated three major versions of phpunit in this pull, not surprising the method signature changed.

Reply:
/shrug